engine/src/main/java/org/terasology/rendering/dag/nodes/WorldReflectionNode.java

```diff
@@ -100,8 +101,8 @@ public void process() {
         playerCamera.setReflected(false);
 
         GL11.glCullFace(GL11.GL_BACK);
-        bindDisplay();
-        setViewportToSizeOf(sceneOpaque);
+        bindDisplay(); // TODO: remove since its reset by default BindFBO state change
```

Comment:
I was trying to be cautious to not remove anything here because after `bindDisplay()`, `setViewportToSizeOf` is called. I'll be looking into this situation in more detail while focusing adding state changes to nodes.